projects/packages/forms/src/blocks/contact-form/components/jetpack-field-file/index.js

```diff
@@ -8,89 +8,6 @@ import JetpackFieldControls from '../jetpack-field-controls';
 import JetpackFieldLabel from '../jetpack-field-label';
 import { useJetpackFieldStyles } from '../use-jetpack-field-styles';
 
```

Comment:
I can move this file out of it's own folder since it is just one file now. Convention in this package would just be to rename this file jetpack-field-file.js and put it directly under components/

I partly wanted to keep it to ensure a cleaner diff for comparison for now.

Reply:
Ideally, this package would follow general WordPress and core conventions with each block in its own dedicated folder.

This is something that we are actually moving towards via https://github.com/Automattic/jetpack/pull/42889 & #42358

Reply:
That definitely works for me. I'll keep it in its own folder.

Reply:
Sounds like a plan :+1:  

I think it makes more sense if the file field is updated to use complete inner blocks i.e. the Label block too. Then all blocks using the inner blocks structure will have their own folder as per #42890.